splitio/storage/pluggable.py

```diff
@@ -0,0 +1,195 @@
+
+    def get(self, split_name):
+        """
+        Retrieve a split.
+
+        :param split_name: Name of the feature to fetch.
+        :type split_name: str
+
+        :rtype: splitio.models.splits.Split
+        """
+        split = self._pluggable_adapter.get(self._prefix + split_name)
+        if not split:
+            return None
+        return splits.from_raw(split)
+
+    def fetch_many(self, split_names):
+        """
+        Retrieve splits.
+
+        :param split_names: Names of the features to fetch.
+        :type split_name: list(str)
+
+        :return: A dict with split objects parsed from queue.
+        :rtype: dict(split_name, splitio.models.splits.Split)
+        """
+        return {split_name: self.get(split_name) for split_name in split_names}
+
+    def put_many(self, splits, change_number):
+        """
+        Store a split.
+
+        :param split: Split object.
+        :type split: splitio.models.split.Split
+        """
+        for split in splits:
+            self.put(split)
+        self._pluggable_adapter.set(self._split_till_prefix, change_number)
+
+    def remove(self, split_name):
+        """
+        Remove a split from storage.
+
+        :param split_name: Name of the feature to remove.
+        :type split_name: str
+
+        :return: True if the split was found and removed. False otherwise.
+        :rtype: bool
+        """
+        split = self.get(split_name)
+        if not split:
+            _LOGGER.warning("Tried to delete nonexistant split %s. Skipping", split_name)
+            return False
+
+        self._pluggable_adapter.delete(self._prefix + split_name)
+        self._pluggable_adapter.decrement(self._traffic_type_prefix + split.traffic_type_name, 1)
+        if self._pluggable_adapter.get(self._traffic_type_prefix + split.traffic_type_name) == 0:
+            self._pluggable_adapter.delete(self._traffic_type_prefix + split.traffic_type_name)
+        return True
+
+    def get_change_number(self):
+        """
+        Retrieve latest split change number.
+
+        :rtype: int
+        """
+        return self._pluggable_adapter.get(self._split_till_prefix)
+
+    def set_change_number(self, new_change_number):
+        """
+        Set the latest change number.
+
+        :param new_change_number: New change number.
+        :type new_change_number: int
+        """
+        self._pluggable_adapter.set(self._split_till_prefix, new_change_number)
+
+    def get_split_names(self):
+        """
+        Retrieve a list of all split names.
+
+        :return: List of split names.
+        :rtype: list(str)
+        """
+        return [split.name for split in self.get_all()]
+
+
+    def get_all(self):
+        """
+        Return all the splits.
+
+        :return: List of all the splits.
+        :rtype: list
+        """
+        return [splits.from_raw(split) for split in self._pluggable_adapter.get_keys_by_prefix(self._prefix)]
```

Comment:
Check here since `get_keys_by_prefix` is returning keys rather than split definitions.

Reply:
that is correct, will fix it.